packages/lib/fs-driver-base.ts

```diff
@@ -152,13 +153,28 @@ export default class FsDriverBase {
 		}
 		let counter = 1;
 
+		// On Windows, ./FiLe.md and ./file.md are equivalent file paths.
+		const caseSensitiveCompare = !shim.isWindows();
+		if (!caseSensitiveCompare) {
+			// Simplify case-insensitive comparison by lowercasing all reserved names
+			// early.
+			reservedNames = reservedNames.map(name => name.toLowerCase());
+		}
```

Comment:
Avoid having a special case here. *Always* use case insensitive comparison so that exports can be safely copied to Windows.